packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -375,6 +375,12 @@ export async function getCommonConfig(wco: WebpackConfigOptions): Promise<Config
           : undefined,
 
       rules: [
+        {
+          test: /\.?(svg|html)$/,
+          // Only process HTML and SVG which are known Angular component resources.
+          resourceQuery: /ngResource/,
```

Comment:
Should we include the `?` in this query (`/?ngResource/`)? I'm wondering if this might pick up other files not intended to be resources (like `import ./routingResource.ts`). Is there some processing which makes sure that user imports don't get picked up here?

I guess the Webpack documentation doesn't include the `?`, so I'm not sure if this is a real issue. If this is intended to look like a URL parameter, then it should technically also accept `&ngResource`, but I'm guessing that's not a real problem?

Reply:
I think adding the `?` might be safer.